cosmwasm/utils.js

```diff
@@ -420,6 +420,28 @@ const makeMultisigProverInstantiateMsg = (config, chainName) => {
     };
 };
 
+const makeAxelarnetGatewayInstantiateMsg = (config, chainName) => {
+    const {
+        axelar: { contracts },
+    } = config;
+    const chainConfig = getChainConfig(config, chainName);
+
+    const { axelarId: chainId } = chainConfig;
+
+    const {
+        Router: { address: routerAddress },
+    } = contracts;
+
+    if (!validateAddress(routerAddress)) {
+        throw new Error('Missing or invalid Router.address in axelar info');
+    }
+
+    return {
+        router_address: routerAddress,
+        chain_name: chainId,
```

Comment:
Lowercase it as well. we don't support upper case chain names. So it'll end up being `axelarnet`